src/pages/index.js

```diff
@@ -42,18 +45,47 @@ const IndexPage = () => {
     return img.find((x) => x.name === 'nologo').childImageSharp.fluid;
   };
 
+  const [favoriteMakers, setFavoriteMakers] = useState(getFavoriteMakers());
+
+  const sortedMakers = sortBy(data.allSitePage.nodes, (n) => !favoriteMakers.includes(n.context.maker.id));
```

Comment:
I wonder if this line that make the CI fails

Reply:
do you mean this error in the CI?

![image](https://user-images.githubusercontent.com/10458061/141072614-fed86d5d-4a96-4a04-a4a4-57398b311404.png)

Reply:
Yes

Reply:
I have debugged, this sculpt cause that issue. Do you want to fix it in the database also?

![image](https://user-images.githubusercontent.com/10458061/141074896-bea35028-14c8-4c29-a8a0-5fc79d32e8ad.png)